IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcRelInterferesElements/DocEntity.xml

```diff
@@ -15,10 +15,10 @@
 			<Documentation></Documentation>
 		</DocAttribute>
 		<DocAttribute Name="InterferenceType" UniqueId="a92075e8-181d-4233-bd6a-e3efed6da830" DefinedType="IfcIdentifier" AttributeFlags="1">
-			<Documentation>Optional identifier that describes the nature of the interference. Examples include  &apos;&apos;Clash&apos;&apos;, &apos;&apos;ProvisionForVoid&apos;&apos; (physical elements), and �Crosses�, �PassesThrough� �PassesOver� �PassesUnder� (spatial elements).</Documentation>
+			<Documentation>Optional identifier that describes the nature of the interference, see details in documentation.</Documentation>
 		</DocAttribute>
 		<DocAttribute Name="ImpliedOrder" UniqueId="57512488-73cf-44cc-995d-d5e84df6ae44" DefinedType="IfcLogical">
-			<Documentation>Logical value indicating whether the interference geometry should be subtracted from the _RelatingElement_ (if TRUE), or whether it should be either subtracted from the _RelatingElement_ or the _RelatedElement_ (if FALSE), or whether no indication can be provided (if UNKNOWN).</Documentation>
+			<Documentation>Logical value indicating if the _RelatingElement_ has priority over the _RelatedElement_. TRUE value imply that relation is oriented from _RelatingElement_ to _RelatedElement_ (in regards of the _InterferenceType_) and that the _InterferenceGeometry_ should be subtracted from the _RelatingElement_. FASLE value imply that relation has no orientation and that _InterferenceGeometry_ may be subtracted from the _RelatingElement_ or the _RelatedElement_.UNKNOWN value imply that no information about the orientation is provided</Documentation>
```

Comment:
Spatial structure elements are not supposed to intersect. That is kind of their purpose, reserving space. If subtracting anything, does the dataset have to feature the subtracted geometry, or are both elements supposed to have original geometry (e.g. IfcRelVoidsElement)?

Reply:
Replied in #39 to have better view by participants.

Reply:
This documentation does not fit anymore or am I missing something?